Bottom: 5ed64bd20d8abe41bc057ec0d608437a25f45638
Top:    e7a0112f8ea65becd91cf71f7fef15f9f322e949
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:52 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
